backend/guides/code.md

```diff
@@ -47,18 +242,18 @@ This practice is sometimes referred to as [💋 KISS](https://en.wikipedia.org/w
 
 ## Prefer libraries
 
-Prefer libraries above your code - you transfer the responsibility to the libraries, that usually are well tested
+Prefer libraries above your code - you transfer the responsibility to the libraries, that usually are well tested. 
+
+The current list of our preferred tools and libraries is reflected in our [Tech radar](https://radar.thoughtworks.com/?sheetId=https://docs.google.com/spreadsheets/d/11FExRfLBnZCM24c2MMPp8SN7U_VqaknZRHnTsAeDYt4/edit&sheetName=Current).
 
 Get to know our libs!
 
-- Logging. https://github.com/AckeeCZ/cosmas
-- Configuration. https://github.com/AckeeCZ/configuru
-- User notifications. https://github.com/AckeeCZ/enmail
-- Message queues. https://github.com/AckeeCZ/fuqu
-- MySQL database. https://github.com/AckeeCZ/databless
-- Authentication. https://github.com/AckeeCZ/authist
-- HTTP Server. https://github.com/AckeeCZ/unicore
-- ACL. https://github.com/AckeeCZ/axesor
+- Node app setup tool https://github.com/AckeeCZ/create-node-app
+- Configuration https://github.com/AckeeCZ/configuru
+- HTTP Server https://github.com/AckeeCZ/unicore
```

Comment:
This one is already derecated.

Btw maybe I'd keep just the radar? It's more comprehensive and we can then treat is as single source of truth.